src/core/RungeKutta.cpp

```diff
@@ -0,0 +1,90 @@
+//             k1[i] = dt * particles[i].velocity;
+//             l1[i] = dt * (1.0 / particles[i].mass) * particles[i].force;
+//         }
+//
+//         // k2 and l2
+//         for (size_t i = 0; i < particles.size(); i++) {
+//             particles[i].position = position0[i] + 0.5 * k1[i];
+//             particles[i].velocity = velocity0[i] + 0.5 * l1[i];
+//         }
+//         compute_forces();
+//         for (size_t i = 0; i < particles.size(); i++) {
+//             k2[i] = dt * particles[i].velocity;
+//             l2[i] = dt * (1.0 / particles[i].mass) * particles[i].force;
+//         }
+//
+//         // k3 and l3
+//         for (size_t i = 0; i < particles.size(); i++) {
+//             particles[i].position = position0[i] + 0.5 * k2[i];
+//             particles[i].velocity = velocity0[i] + 0.5 * l2[i];
+//         }
+//         compute_forces();
+//         for (size_t i = 0; i < particles.size(); i++) {
+//             k3[i] = dt * particles[i].velocity;
+//             l3[i] = dt * (1.0 / particles[i].mass) * particles[i].force;
+//         }
+//
+//         // k4 and l4
+//         for (size_t i = 0; i < particles.size(); i++) {
+//             particles[i].position = position0[i] + k3[i];
+//             particles[i].velocity = velocity0[i] + l3[i];
+//         }
+//         compute_forces();
+//         for (size_t i = 0; i < particles.size(); i++) {
+//             k4[i] = dt * particles[i].velocity;
+//             l4[i] = dt * (1.0 / particles[i].mass) * particles[i].force;
+//         }
+//
+//         // Update positions and velocities
+//         for (size_t i = 0; i < particles.size(); i++) {
+//             particles[i].position = position0[i] + 1.0 / 6 * (k1[i] + 2.0 * k2[i] + 2.0 * k3[i] + k4[i]);
+//             particles[i].velocity = velocity0[i] + 1.0 / 6 * (l1[i] + 2.0 * l2[i] + 2.0 * l3[i] + l4[i]);
+//             SPDLOG_TRACE("Updated position: [{},{},{}]", particles[i].position[0], particles[i].position[1],
+//                          particles[i].position[2]);
+//             SPDLOG_TRACE("Updated velocity: [{},{},{}]", particles[i].velocity[0], particles[i].velocity[1],
+//                          particles[i].velocity[2]);
+//             SPDLOG_TRACE("Updated force: [{},{},{}]", particles[i].force[0], particles[i].force[1],
+//                          particles[i].force[2]);
+//         }
+//     }
+//
+//     void RungeKutta::compute_forces() const {
+//         // Reset forces
+//         for (auto& p : particles) {
+//             p.force = {0.0, 0.0, 0.0};
+//         }
+//
+//         for (size_t i = 0; i < particles.size(); i++) {
+//             for (size_t j = i + 1; j < particles.size(); j++) {
+//                 vec3 force = force_func(particles[i], particles[j]);
+//                 particles[i].force = particles[i].force - force;
+//                 particles[j].force = particles[j].force + force;
+//             }
+//         }
+//     }
+// }  // namespace md::Integrator
```

Comment:
we kept it since we wanted to perhaps re-implement it if we find the time, but yes, deleting it and going back to older commits is probably the wiser choice